mlflow/models/evaluation/base.py

```diff
@@ -161,6 +354,7 @@ def make_metric(
     metric_details=None,
     metric_metadata=None,
     genai_metric_args=None,
+    strict_signature=False,
```

Comment:
nit: Is it possible to determine this by function signature? If not, can we rename it to `standard_signature` or sth? ("strict" sounds like, if it is `false`, the function does not validate signature).